src/Land/Model/soil_model.jl

```diff
@@ -22,6 +22,24 @@ Base.@kwdef struct SoilParamFunctions{FT} <: AbstractSoilParameterFunctions{FT}
     Ksat::FT = FT(NaN)
```

Comment:
would you mind putting units on Ksat and on Specific storage? :) my bad!

Reply:
ok done!